### ISSUE TITLE
Project including "System.Management" nuget can not be used by other project(dynamic load this project), because "System.Management" causes an exception: FileNotFoundException

### ISSUE THREAD

**Issue by ehnap** [dotnet/corefx#42707]
[TestManagementLoad.zip](https://github.com/dotnet/corefx/files/3886032/TestManagementLoad.zip)

System.Management is 4.6.0

Maybe the same question: [#39975](https://github.com/dotnet/corefx/issues/39975)

I write a demo.

- Project ManagementProxy add "System.Management" nuget and use it.
- Project TestManagementLoad loads Project ManagementProxy. 

Run it. An exception occurred.

**System.Reflection.TargetInvocationException:“Exception has been thrown by the target of an invocation.”**
Inner exception
**FileNotFoundException: Could not load file or assembly 'System.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'**

**Comment by danmosemsft** [comment]
Does it help if you make TestManagementLoad have a direct package reference too?

**Comment by ehnap** [comment]
@danmosemsft  Yes. If I make TestManagementLoad have a direct package reference too, I can use it normally.  But actually, maybe TestManagementLoad don't know the implementation of ManagementProxy. So I think there is no need to let TestManagementLoad have a direct package reference.

**Comment by danmosemsft** [comment]
Is nunit involved?

**Comment by ehnap** [comment]
@danmosemsft No. I don't use nunit. 

In [#39975](https://github.com/dotnet/corefx/issues/39975#issuecomment-536237064):

> @ViktorHofer I'm also having this issue, not with NUnit, but when dynamically loading my own assembly that uses System.Management. Can you please explain or link to how "This can be done with an AssemblyLoadContext"? I've spent a couple hours on Google trying to figure it out, but either there's not much info available or I don't know what I should be searching for. (I'm new to .NET Core.) Thanks!

He maybe meet the same problem. And without nunit, too.

**Comment by danmosemsft** [comment]
Perhaps @joperezr  can help she light on these issues
